### PR TITLE
Fix spelling errors and configs

### PR DESCRIPTION
**Pull request by mohsen1**

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -9,10 +9,17 @@
     "language": "en",
     "useGitignore": true,
     "words": [
+        "bluelightbulb",
+        "Caartaa",
         "codebases",
+        "coderepair",
+        "dtos",
```
Comment (mohsen1) [inline cspell.json:16]: I'm not sure about this. What is a DTO?

Comment (grzpab) [reply cspell.json:16]: that's a data transfer object

File: cspell.json [hunk 2]
```diff
@@ -9,10 +9,17 @@
...
+        "gppd",
```
Comment (grzpab) [inline cspell.json:17]: where did you get that from?

File: src/leftRightHashes/leftRightHashSetManager.ts [hunk 3]
```diff
@@ -83,15 +83,15 @@ export class LeftRightHashSetManager<L extends string, R extends string> {
...
 
 		for (const leftRightHash of this._set.keys()) {
 			if (leftRightHash.endsWith(rightHash)) {
-				deletables.push(leftRightHash);
+				canBeDeleted.push(leftRightHash);
 			}
 		}
 
-		for (const deletable of deletables) {
+		for (const deletable of canBeDeleted) {
```
Comment (grzpab) [inline src/leftRightHashes/leftRightHashSetManager.ts:94]: `const hash of canBeDeleted`

File: cspell.json (hunk 2 above)
Comment (mohsen1) [reply cspell.json:17]: https://github.com/intuita-inc/intuita-vscode-extension/blob/60fdf19d67020137688562ef10293d39f9a49a7a/test/classifier/diagnostics.txt#L2

File: cspell.json (hunk 1 above)
Comment (grzpab) [reply cspell.json]: sure, please do

File: cspell.json (hunk 2 above)
Comment (grzpab) [reply cspell.json:16]: I see, didn't know we were checking txt files ;) all good

**Review by gregintuitaio — APPROVED** [review]

**Review by alex-from-intuita — APPROVED** [review]

**Pull request merged**